applications/asset_tracker_v2/tests/debug_module/src/debug_module_test.c

```diff
@@ -70,13 +70,15 @@ void setUp(void)
 	cmock_watchdog_app_Init();
 	cmock_modules_common_Init();
 	cmock_app_event_manager_Init();
+	cmock_app_event_manager_priv_Init();
 }
```

Comment:
Apparently init and verify of mocks happens automatically. So the `setUp` and `tearDown` functions can be removed.
cc @balaji-nordic

Reply:
There will be a separate PR to remove those.